src/services/common/enums/http/http-code.enum.ts

```diff
@@ -0,0 +1,10 @@
+enum HttpCode {
```

Comment:
For what we have http codes? I think, they are used on server side

Reply:
I find this error handler:

**class httpError extends Error {
  status: httpCode

  constructor({
    status = httpCode.INTERNAL_SERVER_ERROR,
    message = DEFAULT_MESSAGE
  } = {}) {
    super(message)
    this.status = status
    this.name = customExceptionName.HTTP_ERROR
  }
}**

It usefull in httpService for throwing error. I don't now maybe everything broken when i delete it )